code/game/objects/structures/crates_lockers/closets/secure/engineering_lockers.dm

```diff
@@ -119,6 +119,6 @@
 	new /obj/item/watertank/atmos(src)
 	new /obj/item/clothing/suit/fire/atmos(src)
 	new /obj/item/clothing/head/hardhat/atmos(src)
-	new /obj/item/clothing/glasses/meson/engine/tray(src)
+	new /obj/item/clothing/glasses/meson/engine(src)
```

Comment:
Can you add the tray ones to atmos vendors? They have a longer-tray sight range than the normal engi ones

Reply:
Adds 4 in the Engi-vend to compensate